godel_process_path_generation/src/process_path_generator_node.cpp

```diff
@@ -26,7 +26,7 @@
 #include <boost/tuple/tuple.hpp>
 #include <godel_process_path_generation/utils.h>
 #include <godel_process_path_generation/VisualizeBlendingPlan.h>
-#include <godel_msgs/ProcessPlanning.h>
+#include <godel_msgs/ProcessPlanningAction.h>
```

Comment:
What's going on in this file? Changes just to keep it compiling? Is this code dead?